instrumentation/armeria-1.0/auto/armeria-1.0-auto.gradle

```diff
@@ -22,3 +22,9 @@ dependencies {
 
   testImplementation project(':instrumentation:armeria-1.0:testing')
 }
+
+tasks.withType(Test) {
+  // Armeria has its own mechanism for context propagation which we don't want our automatic context
+  // propagation mechanism to interfere with.
+  jvmArgs '-Dotel.integration.java_concurrent.enabled=false'
```

Comment:
How this will work in production where `java_concurrent` instrumentation IS enabled by default? Don't you want to test in realistic env?

Reply:
Yeah I don't have a good story for this yet - I tried to inject the context storage override and play with this flag disabled but the bootstrap classloader issue I ran into is problematic. I may need to just instrument the context methods themselves. I will follow up for it.

Interestingly, even with concurrent instrumentation, my unit test fails - I don't do anything particularly weird in it I think, just call `CompletableFuture` :(